apps/registration-api/RegistrationApi.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const { exec } = require("child_process");
+const { join } = require("path");
+const cors = require("cors");
+const axios = require("axios");
+const fs = require("fs");
+var async = require("async");
+const tmp = require("tmp");
+const archiver = require("archiver");
+
+const app = express();
+const PORT = process.env.PORT || 5000;
+
+//take the body of incoming POST requests and turn it into a JavaScript object
+app.use(express.json());
+app.use(cors());
+app.use(express.static(join(__dirname, "static")));
+
+var BH = "";
```

Comment:
Use more informative names such as `beehive` (so that someone new to the code knows what this is).  But, there is also no reason to declare these variables outside of the scope of the functions below.